.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,29 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Check Prettier
+
+# Controls when the action will run. Triggers the workflow on pull request
+on:
+  pull_request:
+    # branches: [ master ]
+    paths:
+      - 'dist/**'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+    prettier:
+        runs-on: ubuntu-latest
+        steps:
+          - uses: actions/checkout@v2
+            with:
+              ref: ${{ github.head_ref }}
+          - uses: actions/setup-node@v1
+            with:
+              node-version: "12.x"
+          - name: Install
+            run: npm install
+            env:
+                CI: true
+
+          - name: Check prettier
+            run: npm run check-prettier
```

Comment:
change this to `npm run clean-css` as this is the command that will minify the file.